addons/hr_contract/security/security.xml

```diff
@@ -6,10 +6,16 @@
             <field name="sequence">10</field>
         </record>
 
+        <record id="hr_contract.group_hr_contract_employee_manager" model="res.groups">
+            <field name="name">Employee Manager</field>
+            <field name="category_id" ref="base.module_category_hidden"/>
```

Comment:
Shouldn't it be `module_category_human_resources_contracts` instead of hidden ?

Reply:
I have hidden it as it is a group implied by hr_payroll.group_hr_payroll_employee_manager to allow viewing contracts with perm_read only. It allows the user to only select Manager group for hr_payroll and have access to the contracts when opening the payroll app. Being able to see contracts without the hr_payroll app could also make sense, in this case, I can add it to `module_category_human_resources_contracts`